examples/Notebooks/flopy3_ModelCheckerExample.ipynb

```diff
@@ -432,7 +425,7 @@
       "    RCH package: Mean R/T ratio < checker warning threshold of 2e-08 for 1 stress periods\n",
       "    1 instance of \r",
       "    RCH package: Variable NRCHOP set to value other than 3\n",
-      "  see /tmp/tmp48abtm22/checksummary.csv for details.\n",
+      "  see C:\\Users\\jlarsen\\AppData\\Local\\Temp\\7\\tmprdhftahz\\checksummary.csv for details.\n",
```

Comment:
Is this still a problem?

Reply:
@langevin-usgs. Yes, I missed that one and a couple of others. They've been fixed now.